### PR TITLE
Download a version of authfile-update with …

### PR DESCRIPTION
**Pull request by matyasselmeci**
… https://github.com/opensciencegrid/xcache/pull/131 if necessary

This assumes the xcache 3.2.0 RPMs will have it included.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,6 +43,13 @@ RUN yum install -y \
         gperftools-devel && \
     yum clean all --enablerepo=* && rm -rf /var/cache/yum/
 
+# If necessary, download the authfile-update version that can do per-instance authfiles
+RUN \
+    if /usr/local/bin/pkg-cmp-lt.sh xcache 3.2.0; then \
+        curl -Lo /usr/libexec/xcache/authfile-update https://raw.githubusercontent.com/opensciencegrid/xcache/V3-branch/src/authfile-update && \
```
Comment (edquist) [inline Dockerfile:49]: Right into /usr, ey?  Bold!

So xcache gets installed from yum out of `BASE_YUM_REPO`, but `authfile-update` is always taken from the tip of `V3-branch` (not necessarily part of a release) ?

Comment (matyasselmeci) [reply Dockerfile:49]: Yup. More specifically, `authfile-update` will be taken from the tip of `V3-branch` until `xcache-3.2.0` or newer is in `BASE_YUM_REPO`.

Comment (edquist) [reply Dockerfile:49]: Okie doke then!

File: Dockerfile [hunk 2]
```diff
@@ -43,6 +43,13 @@ RUN yum install -y \
...
+        chmod +x /usr/libexec/xcache/authfile-update; \
```
Comment (edquist) [inline Dockerfile:50]: Just to be explicit i might suggest `chmod 755` here as the `w` bits depend on the umask when the curl command is run (you might end up with 755 or 775) ... but that's pretty minor and unlikely to cause problems either way, so up to you however you like it best.

**Review by edquist — APPROVED** [review]
one optional comment but lgtm

File: Dockerfile (hunk 2 above)
Comment (matyasselmeci) [reply Dockerfile]: sure, thanks

**Pull request merged**